sdk/contentsafety/azure-ai-contentsafety/src/samples/java/com/azure/ai/contentsafety/ManageTextBlocklist.java

```diff
@@ -4,15 +4,7 @@
 
 package com.azure.ai.contentsafety;
 
-import com.azure.ai.contentsafety.models.TextBlockItem;
-import com.azure.ai.contentsafety.models.TextBlocklistMatchResult;
-import com.azure.ai.contentsafety.models.AddBlockItemsResult;
-import com.azure.ai.contentsafety.models.TextBlockItemInfo;
-import com.azure.ai.contentsafety.models.RemoveBlockItemsOptions;
-import com.azure.ai.contentsafety.models.AnalyzeTextOptions;
-import com.azure.ai.contentsafety.models.AnalyzeTextResult;
-import com.azure.ai.contentsafety.models.TextBlocklist;
-import com.azure.ai.contentsafety.models.AddBlockItemsOptions;
+import com.azure.ai.contentsafety.models.*;
```

Comment:
Do not use `import *`. Please fix all of them.

Check your IDE and disable that.